catalog/confluence/catalog-confluence-source/src/main/java/org/codice/ddf/confluence/source/ConfluenceSource.java

```diff
@@ -124,12 +123,18 @@ public ConfluenceSource(FilterAdapter adapter, EncryptionService encryptionServi
     }
 
     public void init() {
+        if (StringUtils.isBlank(endpointUrl)) {
```

Comment:
Yes, this is because the init gets called from setters for username, password and endpointUrl. So when endpoint URL might not be set yet when the username is set. If there is no url set after everything has been initialized the source will just be unavailable.